components/engine_traits/src/util.rs

```diff
@@ -1,5 +1,13 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
```

Comment:
Can you add more docs to describe how it works and how should it be used?

Reply:
Added some comments for `SequenceNumber` and `SequenceNumberWindow`